common/types.go

```diff
@@ -184,6 +184,10 @@ func (h Hash) Value() (driver.Value, error) {
 	return h[:], nil
 }
 
+func (a Address) IsInContext() bool {
```

Comment:
I thought we decided to use `IsInChainScope()` instead? They're logically equivalent, so we shouldn't have two.

Reply:
Removed